README.md

```diff
@@ -1,90 +1,88 @@
-# Instructions for using the BSEC Arduino Library in Arduino 1.8.5
+# Instructions for using the BSEC Arduino Library in Arduino 1.8.8
 
-Until the Arduino IDE natively supports pre-compiled libraries, the following steps/hacks will need to be followed to integrate the BSEC library into your project.
+The following steps/hacks will need to be followed to integrate the BSEC library into your project.
 
 ## Installation and getting started
 
 ### 1. Install the latest Arduino IDE
 
-As of this publication, the latest Arduino IDE 1.8.5 can be downloaded from this [link](https://www.arduino.cc/download_handler.php)
+As of this publication, the latest Arduino IDE 1.8.8 can be downloaded from this [link](https://www.arduino.cc/download_handler.php)
 
 ### 2. Install the BSEC library
 
 Either download this library as a zip and import it into the Arduino IDE. Refer to [this](https://www.arduino.cc/en/Guide/Libraries) guide on how to import libraries.
 
-### 3. Replace the arduino-builder
-
-The `arduino-builder-PR219` now has the ability to select the appropriate pre-compiled library from the library folder.
-
-Replace the arduino-builder executable from the Arduino installation folder with the appropriate file based on your OS from this link [arduino-builder-219.zip](http://downloads.arduino.cc/PR/arduino-builder/arduino-builder-219.zip).
-
-### 4. Modify the platform.txt file
+### 3. Modify the platform.txt file
 
 If you have already used the previous example code remove the linker flag `-libalgobsec` in the platform.txt file.
 
-The arduino-builder passes the linker flags under `compiler.c.elf.extra_flags`. Most platform.txt files in the combine recipe place this previously unused variable in the start of the link recipe whereas it should be near the end along with the other pre-compiled libraries flags.
+The arduino-builder passes the linker flags under `{compiler.libraries.ldflags}`. Open you platform.txt and add this near the end of your recipe.c.combine.pattern.
```

Comment:
Open **your**

Reply:
Also, this variable needs to be added in the '# These can be overridden in platform.local.txt' section as 'compiler.libraries.ldflags='